docs/config-cli-lint-mode.md

```diff
@@ -8,22 +8,22 @@ description: Cli lint mode can be list_of_files, project or files
 
 # CLI lint mode
 
-Each linter has a lint mode by default, visible in its MegaLinter documentation ([example](https://megalinter.io/latest/descriptors/repository_trivy/#how-the-linting-is-performed)):
+Each linter is pre-configured to use a default lint mode, which are visible in the MegaLinter documentation ([example](https://megalinter.io/latest/descriptors/repository_trivy/#how-the-linting-is-performed)). The possible values are:
 
-- `list_of_files`: All files are sent in single call to the linter
-- `project`: The linter is called from the root of the project, without specifying any file name
-- `file`: The linter is called once by file (so the performances may not be very good)
+- `list_of_files`: The linter is called only once, and passed a list of all the files to be processed
+- `project`: The linter is responsible to scan all the files in the repository, as no filenames are provided it
```

Comment:
Personally, I prefer the original wording of this line since many linters will be configured to ignore certain files. In the original copy, I would replace "file name" with "filenames" though.

Reply:
@Kurt-von-Laven fixed